### PR TITLE
General website updates: Issue 178

### PR DESCRIPTION
**Pull request by SamFritz**
This PR implements the following changes for[ Issue 178](https://github.com/archivesunleashed/archivesunleashed.org/issues/178):

Getting Started Page - The Cloud
* Captured and update screen shots

Archives Unleashed Cloud
* Captured and update screen shots
* Update prose a bit to reflect recent work (new graphs)
* Minor prose fixes

Get Involved
* Packaged recent newsletter as PDF
* Updated all newsletter links (MD formatting)

Publication Page
* Reworked top subheading: Archives Unleashed Materials
* Packaged community report as PDF
* Uploaded citation + link for community report
* Uploaded citation + link for datathon report
* Added COVID DL article citation + link
* Added JDCL long paper citation + AMC link (kept pre-print link)
* Added JCDL poster w/ citation + York pre-print  + PDF demo

Local testing shows images/opens links. @ianmilligan1, no rush on this, but if you could do a quick review to make sure things look good on your end. Thanks!

### CODE REVIEW

File: content/cloud/index.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Core features of the Archives Unleashed Cloud include:
 * Syncing [Archive-It](https://archive-it.org) collections;
 * Ingesting Archive-It collections via WASAPI to the Cloud;
 * Creating a network graph, domain distribution list, full text, and full text by domain derivatives, and making each available for download;
-* Providing an in-browser network diagram to see major nodes and connections within your collection.
+* Providing an in-browser diagrams to highlight a collection's crawl frequency, interactive network graph (to see major nodes and connections within your collection), and domains graph.
```
Comment (ianmilligan1) [inline content/cloud/index.md:26]: Watch tense - maybe drop "an" so it reads "Providing in-browser diagrams..."

File: content/publications/index.md [hunk 2]
```diff
@@ -6,18 +6,17 @@ weight: 48
 
 The Archives Unleashed project has a body of published research.
 
-## Datathon Materials
+## Archives Unleashed Materials
```
Comment (ianmilligan1) [inline content/publications/index.md:9]: Maybe a bit more specific - "Archives Unleashed Project Materials"?

File: content/publications/index.md [hunk 3]
```diff
@@ -6,18 +6,17 @@ weight: 48
...
-
 * Ian Milligan, Nick Ruest, Jimmy Lin, and Samantha Fritz. "Archives Unleashed New York Datathon: Introduction." _Archives Unleashed NY Datathon_, March 2020, Online. [[Datathon Slides](/images/AU-NY-IntroSlides-reduced.pdf")]
 
-## Preprints
-
-* Nick Ruest, Jimmy Lin, Ian Milligan, and Samantha Fritz. "The Archives Unleashed Project: Technology, Process, and Community to Improve Scholarly Access to Web Archives." arXiv:2001.05399, January 2020. [[link](https://arxiv.org/abs/2001.05399)]
 
 ## Journal Articles
 
+* Samantha Fritz, Ian Milligan, Nick Ruest, and Jimmy Lin. "Building community at distance: a datathon during COVID-19." _Digital Library Perspectives_, 2020. https://doi.org/10.1108/DLP-04-2020-0024
```
Comment (ianmilligan1) [inline content/publications/index.md:19]: For consistency, I'd put the DOI in a link

i.e. `[[link](https://doi.org/10.1108/DLP-04-2020-0024)]`

File: content/publications/index.md [hunk 4]
```diff
@@ -37,6 +37,7 @@ The Archives Unleashed project has a body of published research.
 
 ## Peer-Reviewed Posters and Demonstrations
 
+* Tobi Adewoye, Xiao Han, Nick Ruest, Ian Milligan, Samantha Fritz, and Jimmy Lin. 2020. Content-Based Exploration of Archival Images Using Neural Networks. In Proceedings of the ACM/IEEE Joint Conference on Digital Libraries in 2020 (JCDL ’20). Association for Computing Machinery, New York, NY, USA, 489–490. DOI: [https://doi.org/10.1145/3383583.3398577](https://doi.org/10.1145/3383583.3398577). [[preprint](https://yorkspace.library.yorku.ca/xmlui/bitstream/handle/10315/37504/image-search.pdf?sequence=1&isAllowed=y)] [[Poster Demo](/images/jcdl-2020-poster.pdf)] [[Video Summary](https://youtu.be/TQdZbasiI2I)]
```
Comment (ianmilligan1) [inline content/publications/index.md:40]: Funky stuff with the DOI when rendered there (comes out as a fraction I think!). Anyways, for consistency, let's change the DOI to `[[link](https://doi.org/10.1145/3383583.3398577)]`

File: content/publications/index.md [hunk 5]
```diff
@@ -28,6 +27,7 @@ The Archives Unleashed project has a body of published research.
 
 ## Peer-Reviewed Conference Publications
 
+* Nick Ruest, Jimmy Lin, Ian Milligan, and Samantha Fritz. 2020. The Archives Unleashed Project: Technology, Process, and Community to Improve Scholarly Access to Web Archives. In Proceedings of the ACM/IEEE Joint Conference on Digital Libraries in 2020 (JCDL ’20). Association for Computing Machinery, New York, NY, USA, 157–166. DOI: [https://doi.org/10.1145/3383583.3398513](https://dl.acm.org/doi/10.1145/3383583.3398513). [[preprint](https://arxiv.org/abs/2001.05399)] [[Presentation](https://youtu.be/TaVE1cU_EmQ)]
```
Comment (ianmilligan1) [inline content/publications/index.md:30]: Funky stuff with the DOI here too! Change to

`[[link](https://doi.org/10.1145/3383583.3398513)]`

File: content/publications/index.md (hunk 2 above)
Comment (SamFritz) [reply content/publications/index.md:9]: for sure, this is a great suggestion!

**Pull request merged**